tap_stripe/__init__.py

```diff
@@ -911,8 +920,13 @@ def sync_event_updates(stream_name, is_sub_stream):
                     or when called through only child stream i.e. when parent is not selected.
     '''
     LOGGER.info("Started syncing event based updates")
-
-    date_window_size = 60 * 60 * 24 # Seconds in a day
+    event_window_size = Context.event_window_size
+    # Reset event_window_size to 30 days if it is greater than 30 because Stripe Event API returns data of the last 30 days only.
+    if event_window_size > 30:
+        event_window_size = 30
+        LOGGER.warning("Using event window size of 30 days for %s stream.", stream_name)
```

Comment:
Update warning and make like code comment.

Reply:
Update warning message like code comment.